composer.json

```diff
@@ -12,7 +12,7 @@
     "require": {
         "php": "^7.0 || ^8.0",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2 || ^1.0",
-        "phpmd/phpmd": "^2.0",
+        "phpmd/phpmd": "^2.0 || 3.x-dev",
```

Comment:
maybe already add ^3.0 too (even if it does not exist, is it really worth another PR?)